docs/_summary.md

```diff
@@ -54,9 +54,12 @@
   * [Bootmagic](feature_bootmagic.md)
   * [Combos](feature_combo)
   * [Command](feature_command.md)
+  * [Debounce API](feature_debuonce.md)
```

Comment:
I think the file you want is `feature_debounce_type.md` (and maybe that should be renamed?). Also, you misspelled `debounce` here. :stuck_out_tongue: 

```suggestion
  * [Debounce API](feature_debounce_type.md)
```